run/check_network_connectivity.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set +x
+set -e
+
+usage() {
+    echo
+    echo "Usage:"
+    echo "$0 [IP_FILE]"
+    echo
+    echo "Expect a file at IP_FILE (read from env, default to first argument) which contains one IP Adress per non empty non '#' starting line."
+    echo "NMAP scan INTERNAL_PORT (read from env, default 9701) on those machines in order to test routing/firewalls."
+    echo
+}
+
+check_connection() {
+    nmap -Pn -n -p $INTERNAL_PORT $1
```

Comment:
Should we check if nmap is installed ? Otherwise point the user how to receive like 

```
We found that you missed a depedencie for this script:

Please run:
apt-get update && apt-get install -y --no-install-recommends nmap
```

Reply:
good point. in terms of documentation I should likely also add a section to the readme

Reply:
:ballot_box_with_check: